README.md

```diff
@@ -66,6 +66,24 @@ The following steps outline the process of adding new variables to the project.
 
         new_variable: {{ new_variable }}
 
+### Removing old variables
+
+Removed variables will be documented in the [changelog](CHANGELOG.md) for each release. Review the [changelog](CHANGELOG.md) to ensure that all on wanted variables are removed from the configuration files.
+
+#### Encrypted
+
+1. Remove the variable form the `CREDENTIAL_CONFIG.yml` file.
+
+2. Insure the variable has been removed from the `bootstrap_credentials.yml`. This should have been done as part of the update.
+
+3. Insure the variable has been removed from all templates. From the repo's root grep for instances of the variable. This should be an empty return. `grep -rw <old_variable> *`
```

Comment:
Might be easier to copy/paste the `grep` command if it was on a new line. Would also be good to keep some consistency for code blocks e.g. new line and indent command. See other sections of the README for examples